compiler-rt/test/asan/TestCases/Windows/defaultlibs_check.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang_cl_asan -Od %s -Fe%t /link /WX
+// RUN: %clang_cl_asan -Od %s -Fe%t /link /WX /INFERASANLIBS:DEBUG
+
+// Link with /WX under each configuration to ensure there are
+// no warnings (ex: defaultlib mismatch, pragma detect mismatch)
+// when linking.
+

```

Comment:
it's a new test?
new patch?

Reply:
This is a sorta nop test to identify problems with how the linker picks asan libs. It should work fine if added to main, but maybe won't be that useful, and I'm OK with dropping it.

Reply:
I just removed it, I'll include it with all the other tests that actually require further changes.